packages/certifi/meta.yaml

```
@@ -1,11 +1,11 @@
 package:
   name: certifi
-  version: 2024.2.2
+  version: 2024.7.4
   top-level:
     - certifi
 source:
   url: https://files.pythonhosted.org/packages/ba/06/a07f096c664aeb9f01624f858c3add0a4e913d6c96257acb4fce61e7de14/certifi-2024.2.2-py3-none-any.whl
```

```suggestion
  url: https://files.pythonhosted.org/packages/1c/d5/c84e1a17bf61d4df64ca866a1c9a913874b4e9bdc131ec689a0ad013fb36/certifi-2024.7.4-py3-none-any.whl
```

Forgot to update the URL